doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,72 @@
+..
```

Comment:
All of the boilerplate comments should be removed as they are just distracting.

Reply:
fixed